apps/tsilhqotin-language-hub/src/styles.css

```diff
@@ -486,3 +486,60 @@ h2 {
 .language {
     color: #000000c6;
 }
+
+.top-to-btm {
+    position: relative !important;
```

Comment:
Sonar cloud does not like the `!important` in a handful of circumstances. Please look into what is going on there.